test/controllers/stages_controller_test.rb

```diff
@@ -396,4 +402,26 @@ def clone(method, format, extra = {})
       end
     end
   end
+
+  as_an_admin do
+    describe '#new' do
+      it 'can alter `does not deploy code`' do
+        get :new, params: {project_id: subject.project.to_param}
+        assert_select "#stage_no_code_deployed" do |input|
+          refute input.attr("disabled").present?
+        end
```

Comment:
copy-pasted, put into a method so we can assert true/false and don't get accidental diffs

Reply:
like `assert stage_no_code_deployed_disabled` / `refute stage_no_code_deployed_disabled`